pom.xml

```diff
@@ -98,6 +98,13 @@
       <artifactId>metrics</artifactId>
       <version>3.0.0</version>
     </dependency>
+    <!-- test dependencies -->
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
+      <version>4.4.1</version>
+      <scope>test</scope>
```

Comment:
@aheritier Please see [JENKINS-35445](https://issues.jenkins-ci.org/browse/JENKINS-35445). This specific dependency was causing problems in the `hpi` packaging for some dependency graphs, so it has been shaded in the JTH.

If the dependency does not appear on any trail of the compile dependencies, :+1: